src/static/js/templates/prepare-worksheets/input-graded.hbs

```diff
@@ -0,0 +1,15 @@
+<section class="input-graded block__border-bottom u-mb20">
+  <div class="u-mt20 u-mb20">
+    <section class="input-with-btns">
+      <div class="input-with-btns_btns">
+        <button class="btn btn__grouped-first {{#if high_grade}}active{{/if}}">High</button>
+        <button class="btn btn__grouped {{#if medium_grade}}active{{/if}}">Med</button>
+        <button class="btn btn__grouped-last {{#if low_grade}}active{{/if}}">Low</button>
+        <span class="btn-delete cf-form_input-icon cf-icon cf-icon-delete-round" role="button" title="Delete"></span>
+      </div>
+      <div class="input-with-btns_input">
+        <input type="text" value="{{input_value}}" placeholder="Fill in your own goal">
```

Comment:
Sounds good :+1: Alternatively, I had "Lifestyle goal" and "Priority level" inside `<label>`'s originally, that would work also, right? 

Question about the priority label—Since the `for` attribute of a label means the label can be associated with a single form control, should a duplicate label be created for each button, or can screen readers handle something like, e.g.:

```
<label><span class='.u-visually-hidden'>Priority level</span>
    <button class="btn btn__grouped-first">High</button>
    <button class="btn btn__grouped">Med</button>
    <button class="btn btn__grouped-last">Low</button>
</label>
```